test/access-token.test.js

```diff
@@ -32,28 +32,27 @@ describe('loopback.token(options)', function() {
   });
 
   it('should not search default keys when searchDefaultTokenKeys is false',
-  function(done) {
-    var tokenId = this.token.id;
-    var app = createTestApp(
-      this.token,
-      { token: { searchDefaultTokenKeys: false } },
-      done);
-    var agent = request.agent(app);
-
-    // Set the token cookie
-    agent.get('/token').expect(200).end(function(err, res) {
-      if (err) return done(err);
-
-      // Make a request that sets the token in all places searched by default
-      agent.get('/check-access?access_token=' + tokenId)
-        .set('X-Access-Token', tokenId)
-        .set('authorization', tokenId)
-        // Expect 401 because there is no (non-default) place configured where
-        // the middleware should load the token from
-        .expect(401)
-        .end(done);
+    function(done) {
+      var tokenId = this.token.id;
+      var app = createTestApp(
+        this.token, { token: { searchDefaultTokenKeys: false } },
+        done);
+      var agent = request.agent(app);
+
+      // Set the token cookie
+      agent.get('/token').expect(200).end(function(err, res) {
+        if (err) return done(err);
+
+        // Make a request that sets the token in all places searched by default
+        agent.get('/check-access?access_token=' + tokenId)
+          .set('X-Access-Token', tokenId)
+          .set('authorization', tokenId)
+          // Expect 401 because there is no (non-default) place configured where
+          // the middleware should load the token from
+          .expect(401)
+          .end(done);
+      });
```

Comment:
This looks like pure spacing/formatting change to me, unrelated to the changes made in this patch. Could you please revert?